website/docs/reference/snapshot-configs.md

```diff
@@ -80,11 +80,30 @@ snapshots:
 
 <TabItem value="property-yaml">
 
-**Note:** Required snapshot properties _will not_ work when defined in `config` YAML blocks. We recommend that you define these in `dbt_project.yml` or a `config()` block within the snapshot `.sql` file.
```

Comment:
This will still be true for versions before `1.9` - do we want to keep the callout for those earlier versions? Maybe add a recommendation to upgrade?